packages/electrode-archetype-react-app-dev/package.json

```diff
@@ -97,7 +97,6 @@
     "mime": "^1.0.0",
     "mini-css-extract-plugin": "^0.7.0",
     "mkdirp": "^0.5.1",
-    "mocha": "^4.0.0",
```

Comment:
Need to add same changes for electrode-archetype-react-component-dev also

Reply:
good catch! done.